journeys/journeys-service.yaml

```diff
@@ -0,0 +1,197 @@
+openapi: "3.0.1"
+info:
+  title: "SCALE Journeys Service"
+  version: v0.0.1
+  description: This API allows UI's to define journeys associated to a unique identifier supplied. This is used to store the state of progress within the journey. The UI is responsible for supplying a unique identifier within the bounds of itself and the user.
+#  PLACEHOLDERs - to be uncommented when implemented
+  termsOfService: "http://api.crowncommercial.gov.uk/terms/"
+  contact:
+    email: "apiteam@crowncommercial.gov.uk"
+  license:
+    name: "Open Government Licence 3.0"
+    url: "https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/"
+#
+#
+#security:
+#  - OAuth2:
+#    - read
+paths:
+  /journeys:
+    post:
+      description: Creates a new journey defining the number of states and their initial state.
+      tags:
+        - Journeys
+      operationId: create-journey
+      requestBody:
+        content:
+          application/json::
+            schema:
+              $ref: '#/components/schemas/Journey'
+
+      responses:
+        '201':
+            description: TeamID for the newly created Team
+            content:
+                application/json:
+                    schema:
+                        $ref: '#/components/schemas/JourneyId'
+        '401':
```

Comment:
May want a `409 Conflict` or better still `422 Unprocessable Entity` for handling POST requests with an existing `journey-id`

Reply:
Added 422